app/views/idv/doc_auth/recover.html.erb

```diff
@@ -2,11 +2,11 @@
 
 <%= image_tag(asset_url('alert/success.svg'), width: 60) %>
 
-<h1 class='h1 margin-bottom-2 margin-top-4 margin-y-0'>
```

Comment:
multiple h1s!

Reply:
@aduth do you happen to know if this is also part of the recovery flow that is not live in production?  @solipet and I weren't able to easily pull this one up to preview it.  (But if it's part of that old flow we won't stress about it.)

Reply:
Yeah, it would appear so. It looks like these "recover" and "overview" views are the first two steps of the flow:

https://github.com/18F/identity-idp/blob/994b641f84252f9b6be7c4e555aacb873fe4a879/app/services/idv/flows/recovery_flow.rb#L5-L6